fdbserver/workloads/ConsistencyCheck.actor.cpp

```diff
@@ -1933,6 +1938,89 @@ struct ConsistencyCheckWorkload : TestWorkload {
 		return true;
 	}
 
+	// Checks if the blob workers are "correct".
+	// Returns false if ANY of the following
```

Comment:
We could also check for doubling up of blob workers here. (no more than 1 blob worker on a BlobWorkerClass process). Similarly to the "checkForExtraDataStores" check.

Reply:
+1
thanks for the suggestion!

Reply:
Since we don't handle blob manager failures yet, this check will be hard to execute since once a BM dies, it takes the records of BWs to the grave with it. So once a new one is recruited, it starts recruiting BWs unconditionally (i.e. without taking into consideration the existing ones). I left the code we really want as a comment and we can enable it once BM failure is properly handled.